adobe_vipm/airtable/models.py

```diff
@@ -734,3 +738,13 @@ def get_adobe_product_by_marketplace_sku(
 
     AdobeItemModel = get_sku_adobe_mapping_model(AirTableBaseInfo.for_sku_mapping())
     return AdobeItemModel.from_short_id(vendor_external_id)
+
+
+@cache
+def get_adobe_sku(vendor_item_id):
+    return get_adobe_product_by_marketplace_sku(vendor_item_id).sku
+
+
+@lru_cache(maxsize=1900)
```

Comment:
mm... not sure that lru_cache here is the best option. What if data in the table was changed?

Reply:
This method is here only to check if the item is the end of sale. I presume the value of this field will not be changing too frequently and the cache is designed to be valid only for one day. The issue would occur only in the case when the item has been synced on a specific day, then marked as "end of sale" the same day and synced, again on the same day. Let's remember the sync in many cases will be taking place only once per day. The changed will be picked up from the next day onwards, and if we do not cache this value, we will be sending requests to the Airtable over and over again with every sync the item occurs in the currently synced subscriptions.